lib/wallet/integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -1430,7 +1430,7 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
 
         -- transaction setting up reference script
         -- Build:
-        -- $ cardano-cli transaction build \
+        -- \$ cardano-cli transaction build \
```

Comment:
Hi @paolino 

Is this change intentional?

There are also several other similar changes in this file.

Reply:
oh, that's really unrelated. It fixed some warnings, but I can roll them back.

Reply:
removed